doc/releases/release-notes-2.5.rst

```diff
@@ -147,6 +147,16 @@ Architectures
 
 * ARC
 
+  * Fixed execution on HS with one interrupt bank and fast interrupts enabled
```

Comment:
I'd use `ARC HS` as a more typical use of the ARC core name. The same goes to "fast interrupts" which might be useful to also mentioned like `FIRQ`. I.e. something like this:

> Fixed execution on ARC HS with one interrupt bank and fast interrupts (FIRQ) enabled